src/material/icon/icon.scss

```diff
@@ -59,6 +60,33 @@ mat-icon {
 
     .mat-icon-button .mat-icon {
       margin: auto;
+      overflow: visible;
     }
   }
 }
+
+// Makes button icon overflow visible to fix b/250063405
+.mat-mdc-icon-button .mat-icon,
+.mdc-button .mat-icon,
+.mat-mdc-fab-base .mat-icon,
+.mat-toolbar .mat-icon,
+.mat-mdc-tooltip-trigger .mat-icon,
+.mat-mdc-tab .mat-icon,
+.mat-mdc-form-field .mat-icon,
+.mat-grid-tile .mat-icon,
+.mat-mdc-list-item .mat-icon,
+.mat-mdc-list-option .mat-icon {
```

Comment:
I still don't understand why this is targeting only icons inside buttons, form fields etc, instead of all icons.